microcosm_flask/audit.py

```diff
@@ -89,8 +93,8 @@ def audit(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         options = AuditOptions(
-            include_request_body=True,
-            include_response_body=True,
+            include_request_body=DEFAULT_INCLUDE_REQUEST_BODY,
```

Comment:
I am reinterpreting these config parameters as integer size limits (though boolean will still be allowed).

As far as I know, we never configure these today.

The current default is arbitrary.